packages/react/src/GadgetFunctions.ts

```diff
@@ -0,0 +1,73 @@
+import { GadgetRecord, GadgetRecordList, LimitToKnownKeys, VariableOptions } from "@gadgetinc/api-client-core";
```

Comment:
This file was added to api-client-core, but not properly exported from its index. I'm just going to put it in here for now, and come back later to remove it, once we release a version of api-client-core that properly exports its own copy of this file.